src/Symfony/Component/Notifier/Bridge/Clickatell/ClickatellTransportFactory.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Clickatell;
+
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+use Symfony\Component\Notifier\Transport\TransportInterface;
+
+/**
+ * @author Kevin Auvinet <k.auvinet@gmail.com>
+ *
+ * @experimental in 5.3
+ */
+class ClickatellTransportFactory extends AbstractTransportFactory
+{
+    private const SCHEME_NAME = 'clickatell';
```

Comment:
As this scheme name won't change in the future, I don't see a reason to introduce a constant here. And also for consistency with the other `*TransportFactory`s, I would suggest to use the string directly.